lib/text.rb

```diff
@@ -0,0 +1,12 @@
+module Text
+
+  def send_text
+     account_sid = 'ACef87d7dfc16f3cebd55fc9ec9969335a'
+     auth_token = '720f3dadd2aaa05bd816b43b5bd99059'
+     @client = Twilio::REST::Client.new account_sid, auth_token
+     message = @client.account.messages.create(:body => "Thank you! Your order was placed and will be delivered within the next hour",
```

Comment:
Metrics/LineLength: Line is too long. [134/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)